ui/src/__tests__/components/group/__snapshots__/GroupRow.test.js.snap

```diff
@@ -114,6 +114,30 @@ exports[`GroupRow should render 1`] = `
       </svg>
     </span>
   </td>
+  <td
+    class="emotion-6 emotion-3"
+    color="#3570f40D"
+  >
+    <span>
+      <svg
```

Comment:
instead of adding the svg here directly. We maintain a separate icons.js file which has all the relevant svg and use Icon.js to show the icon here. You can check RoleRow.js for reference.

Reply:
@chandrasekhar1996 This is snap file